yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/retrieve/BreakdownResponse.java

```diff
@@ -0,0 +1,54 @@
+package com.yoti.api.client.docs.session.retrieve;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class BreakdownResponse {
+
+    @JsonProperty("sub_check")
+    private String subCheck;
+
+    @JsonProperty("result")
+    private String result;
+
+    @JsonProperty("details")
+    private ArrayNode details;
+
+    public BreakdownResponse() {
+    }
+
+    public BreakdownResponse(String subCheck, String result, ArrayNode details) {
+        this.subCheck = subCheck;
+        this.result = result;
+        this.details = details;
+    }
+
+    public String getSubCheck() {
+        return subCheck;
+    }
+
+    public void setSubCheck(String subCheck) {
```

Comment:
Are public setters needed on `com.yoti.api.client.docs.session.retrieve.*` classes?

Reply:
Fixed in the latest draft